package.json

```diff
@@ -12,8 +12,10 @@
     "start": "NODE_ENV=production node build/server.js",
     "speculate": "speculate",
     "storybook": "start-storybook -p 9001 -c .storybook",
-    "test:e2e": "npx cypress run",
-    "test:e2e:interactive": "npx cypress open",
+    "cypress:run": "npx cypress run",
+    "cypress:interactive": "npx cypress open",
+    "test:e2e": "run-p --race dev cypress:run",
```

Comment:
1. If I'm running my prod env (start) on localhost 7080 then run this it passes with a warning only at the top of the output (asking if I want to use another port). This seems very problematic as the e2e tests ran on the wrong env.

2. Also why not set the e2e tests to run on the prod build, and add the run build step into this command? Don't we always want to run this against the prod build? (This doesn't solve point 1 it only inverts it)

Reply:
I was thinking about adding a different command `"test:e2e:ci": "npm run build && run-p --race dev cypress",` to run against the production build, which can also be used on CI. 

I think it'd be worth keeping the ability to Cypress tests locally against the dev server since you can re-run the tests run as you work.

Reply:
About point 1 - should I always stop whatever is running on port 7080, before running this command?  I can do that with `kill $(lsof -t -i:7080)` but that seems quite destructive.

Reply:
I've made the changes to the branch. I'm not sure of the implications of running a `kill` command on CI. I don't think we would want to run this on CI.